examples/intermediate/07_classification.py

```
@@ -0,0 +1,401 @@
+    "sentiment": "긍정/부정/중립",
+    "confidence": 0.0-1.0,
+    "emotions": ["감정1", "감정2"],
+    "intensity": "약함/보통/강함"
+}}
+"""
+        
+        detailed_reply = llm.ask(detailed_prompt)
+        try:
+            analysis = json.loads(detailed_reply.text)
+            print(f"상세 분석: {analysis}")
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)